versions/0.1.6/README.MD

```diff
@@ -0,0 +1,11 @@
+# Version 0.1.5
```

Comment:
This should indicate that 0.1.6 is the latest version?

Reply:
Fixed